app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,103 @@
  *= require_tree .
  *= require_self
  */
+html {
+    background-color: #343535b0;
+}
+
+.body-text {
+    background-color: rgb(209, 207, 207);
+    padding-bottom: 20px;
+    padding-top: 20px;
+}
+
+li {
+    list-style-type: none;
+    margin: 0 2vw;
+    font-size: 2vh;
+}
+
+nav {
+    width: 100vw;
+    background-color: #D6EFFF;
+}
+
+.header {
+    margin-left: 20px;
+    width: 100%;
+    text-decoration: none;
+    color: white;
+    padding: 2vw;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+.link {
+    text-decoration: none;
+    color: black;
+    padding: 1vw;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+ul {
+    margin: 0;
+    padding-top: 6px;
+    height: 40px;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    top: 50%;
+}
+
+a {
+    color: rgb(15, 15, 15);
+    margin-left: 10px;
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+p {
+    margin-left: 5px;
+    color: black
+}
+
+h2, h3, form, button{
+    margin-left: 5px;
+    padding: 0;
+}
+
+input {
+    width: auto;
+    height: auto;
+    margin-top: 7px;
+    margin-bottom: 10px;
+}
+
+input#post_title {
```

Comment:
In terms of naming classes in css, most would go with dash instead of underscore. So `post-title` instead of `post_title`. Again this is useful just personal preference but I like to stick with whatever the User Portal does.

Reply:
Strangely enough, I actually didn't make that CSS class. Rails made it for me. I have the table posts in a database and one of the columns is title and the input box for the title of a new post just came up to be named this when I inspected the element. Gonna stay like that I reckon.

Reply:
Cool, I've no idea about Ruby or Rails so can't comment on that. It's been mostly React components I've seen this anyway. Anyway, sounds like it should be left if it's like that out of the box 👍

Reply:
Also, look into SMACS (Can't remember what it stands for), Atomic Design and BEM (block, element, modifier) as far as CSS naming is concerned